packages/fgtclb/typo3-category-types/Classes/Loader/CategoryTypeLoader.php

```diff
@@ -58,8 +58,51 @@ public function loadUncached(): array
                 }
                 if (array_key_exists('types', $configArray) && is_array($configArray['types'])) {
                     foreach ($configArray['types'] as $categoryType) {
+                        // Check if the identifier and group of the category type to ensure it is unambiguous
+                        $categoryTypeIdentifier = $categoryType['identifier'] ?? null;
+                        if ($categoryTypeIdentifier === null) {
+                            throw new \Exception(
+                                'Category type identifier has to be defined.',
+                                1678979375330
+                            );
+                        }
+                        $categoryTypeGroup = $categoryType['group'] ?? null;
+                        if ($categoryTypeGroup === null) {
+                            throw new \Exception(
+                                'Category type group has to be defined.',
+                                1678979375330
+                            );
+                        }
+
+                        // Generate an unambiguous array key for the category type
+                        $categoryKey = $categoryTypeGroup . '.' . $categoryTypeIdentifier;
```

Comment:
What if $categoryTypeGroup or $categoryTypeIdentifier is an empty string or not a string at all ?